js/tiles.js

```diff
@@ -515,7 +519,8 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/blood.svg");
           },
         visit: function() {
-            alertInfo("Navy Pink");
+            player.inventory.add(['Rocket', 'Rocket.svg']);
+            alertInfo("You found a Rocket made by Sahil Saha!", "You can go to Space!!!");
```

Comment:
We already have an alert, so why should we add one more?

Reply:
I have already removed the earlier alert

Reply:
Now, 2 alerts appear when the user visits the tile. Please see it and remove this one.